### PR TITLE
Constructing dungeon room

### PR DESCRIPTION
**Pull request by yaboistyryl**
Reproduce 100 random created rooms:

1) pull this branch
2) start program
3) type "skip" where it says press any key.
4) type 2 random names (No data validation)
5) Witness 100 random gen'd rooms.

Will not stay like this, just to prove it's randomly generating.

Design in future is to have 1 room that regens over and over.

### CODE REVIEW

File: adventureGame/AdventureGame/.spyproject/workspace.ini [hunk 1]
```diff
@@ -6,5 +6,5 @@ save_non_project_files = False
 
 [main]
 version = 0.1.0
-recent_files = ['C:\\Users\\mr_ep\\.spyder-py3\\temp.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\obj\\Inventory.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\test\\test_Inventory.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\src\\Main.py']
```
Comment (yaboistyryl) [inline adventureGame/AdventureGame/.spyproject/workspace.ini:9]: This is auto-generated by spyder. Please ignore

File: adventureGame/AdventureGame/obj/Main.py [hunk 2]
```diff
@@ -1,5 +0,0 @@
-"""
```
Comment (yaboistyryl) [inline adventureGame/AdventureGame/obj/Main.py:1]: File was moved to src folder and therefore deleted

File: adventureGame/AdventureGame/obj/DungeonRoom.py [hunk 3]
```diff
@@ -13,26 +16,19 @@ class dungeonRoom:
...
-
+        
         self.setName(_name)
         self.setHasEnemies(_hasEnemies)
         self.setEnemyList(_enemyList)
         self.setHasChest(_hasChest)
         
+        # If debug is enabled, print the instance info.
         if debug == True:
-            print("\nConstructing dungeonRoom...\n")
```
Comment (Aymanchoudhery) [inline adventureGame/AdventureGame/obj/DungeonRoom.py]: Change print statement to:
print("\nConstructing dungeon room...\n")

File: adventureGame/AdventureGame/obj/DungeonRoom.py [hunk 4]
```diff
@@ -52,9 +48,16 @@ def setHasEnemies(self, _hasEnemies):
     
     def setEnemyList(self, _enemyList):
         if type(_enemyList) is list:
-            for enemyListContent in _enemyList:
-                if isinstance(enemyListContent, EnemyClass.Enemy):
-                    self.enemyList = _enemyList
+            if len(_enemyList) == 0:
+                if debug == True:
+                    print("WWARNING: instance of dungeonRoom has an empty enemyList")
```
Comment (Aymanchoudhery) [inline adventureGame/AdventureGame/obj/DungeonRoom.py]: Change print statement to:
print("WARNING: instance of dungeonRoom has an empty enemyList")

File: adventureGame/AdventureGame/src/Main.py [hunk 5]
```diff
@@ -7,20 +7,37 @@
...
     userInput = input()
+    
+    # Clear the terminal screen.
     for i in range(50):
         print("\n")
+    
+    # If the user types "skip", it allows them to skip the introduction story.
     if userInput != "skip":
         introduction()
-    else:
```
Comment (Aymanchoudhery) [inline adventureGame/AdventureGame/src/Main.py:17]: Why two instances if "else:" in same loop? One seems to be obsolete...

**Review by Aymanchoudhery — APPROVED** [review]
Only 1 gramatical error found. There are a few instances of "No new line at end of file"; not sure if these are operation critical.
Those aside, syntax looks in good order.

File: adventureGame/AdventureGame/src/Main.py (hunk 5 above)
Comment (yaboistyryl) [reply adventureGame/AdventureGame/src/Main.py:17]: There is only one instance of "else:". Red indicates that it has been removed, green is added. Just a weird git glitch.

File: adventureGame/AdventureGame/obj/DungeonRoom.py (hunk 4 above)
Comment (yaboistyryl) [reply adventureGame/AdventureGame/obj/DungeonRoom.py]: Fixed.

**Pull request merged**